app/models/workflow.rb

```diff
@@ -37,6 +37,29 @@ def self.new_from_template(template)
     workflow
   end
 
+  # Create a new workflow from a path and attempt to load a manifest on that path.
+  #
+  # @param [String] path A path to use as a non-static template
+  # @return [Workflow] Return a new workflow based on the path
+  def self.new_from_path(path)
+    path = Pathname.new(path).expand_path rescue Pathname.new(path)
+    workflow = Workflow.new
+    workflow.name = path.to_s
```

Comment:
Not sure about using the pathname here, mostly because of #215, but I don't know what else to go with. Considered something like "Created from path" or something. This is the job name that is used when a user creates a job from the path and does not define a custom name or have one imported from a manifest.